src/software/ai/hl/stp/play/stop_play_test.cpp

```diff
@@ -4,13 +4,13 @@
 
 #include "software/simulated_tests/non_terminating_validation_functions/robots_avoid_ball_validation.h"
 #include "software/simulated_tests/non_terminating_validation_functions/robots_slow_down_validation.h"
-#include "software/simulated_tests/simulated_play_test_fixture.h"
+#include "software/simulated_tests/simulated_er_force_sim_play_test_fixture.h"
 #include "software/simulated_tests/validation/validation_function.h"
 #include "software/test_util/test_util.h"
 #include "software/time/duration.h"
 #include "software/world/world.h"
 
-class StopPlayTest : public SimulatedPlayTestFixture
+class StopPlayTest : public SimulatedErForceSimPlayTestFixture
```

Comment:
My experience with porting this test was that it will fail when running with visualizer and pass when I ran it without it. Curious if you also experienced this.
Also, if `halt_play_test` passes for you, you could try porting that over aswell, and we can close #2502.

Reply:
for some reason, it still passes with the visualizer on, on my machine

Reply:
interesting, I may be mistaking then. 
Should we port `halt_play_test` aswell? Or leave that for another PR?

Reply:
ok added halt play test too